### PR TITLE
Implement error dialog of importing files (EXPOSUREAPP-9335)

### PR DESCRIPTION
**Pull request by jurajkusnier**
## Testing
Test Menu > Qr Code scanner > Open Scanner > Datei offnen
Import some corrupted files, images without qr code and/or not valid qr codes and check the error messages

## Jira Ticket
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9335

### CODE REVIEW

**Review by NataliaLemmerth — DISMISSED** [review]
UA approved

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/qrcode/ui/QrCodeScannerViewModel.kt [hunk 1]
```diff
@@ -35,15 +37,20 @@ class QrCodeScannerViewModel @AssistedInject constructor(
...
+        try {
+            when (val parseResult = qrCodeFileParser.decodeQrCodeFile(fileUri)) {
+                is QrCodeFileParser.ParseResult.Failure -> {
+                    Timber.tag(TAG).d(parseResult.exception, "parseResult failed")
+                    result.postValue(Error(error = parseResult.exception))
+                }
+                is QrCodeFileParser.ParseResult.Success -> {
+                    Timber.tag(TAG).d("parseResult=$parseResult")
+                    onScanResult(parseResult.text)
+                }
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/qrcode/ui/QrCodeScannerViewModel.kt:49]: we have to do the same in Submission On behalf flow `OrganizerWarnQrCodeScannerViewModel`

Comment (jurajkusnier) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/qrcode/ui/QrCodeScannerViewModel.kt:49]: `OrganizerWarnQrCodeScannerViewModel` is update

**Review by mtwalli — APPROVED** [review]

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Pull request merged**